spec/rails_helper.rb

```diff
@@ -62,4 +63,12 @@
 
   # serializer_spec_helper
   config.include SerializerSpecHelper, type: :serializer
+
+  # clean up carrierwave uploads
+  # https://github.com/carrierwaveuploader/carrierwave/wiki/How-to:-Cleanup-after-your-Rspec-tests
+  config.after(:all) do
+    if Rails.env.test?
+      FileUtils.rm_rf(Dir["#{Rails.root}/spec/support/uploads"])
+    end
+  end
```

Comment:
My comment was more generic. To me this, and some other carrierwave related things could go in a carrierwave_helper that is only included for certain types of specs that have a particular tag for example. That way this teardown is only executed when a spec that has that tag calls for it, and any additional methods that are helpful in testing uploads could go there as well. But, again, this was the only nitpick comment.